option/internaloption/internaloption.go

```diff
@@ -209,6 +209,27 @@ func (w enableNewAuthLibrary) Apply(o *internal.DialSettings) {
 	o.EnableNewAuthLibrary = bool(w)
 }
 
+// EnableAsyncRefreshDryRun  returns a ClientOption that specifies if libraries in this
```

Comment:
typo: extra space in "EnableAsyncRefreshDryRun  returns"

Reply:
Done